src/rules/no-focused-tests.ts

```diff
@@ -80,16 +98,50 @@ export default createRule({
           callee.object.type === AST_NODE_TYPES.MemberExpression &&
           isCallToTestOnlyFunction(callee.object)
         ) {
+          const calleeObject: TSESTree.MemberExpression = callee.object;
+
           context.report({
             messageId: 'focusedTest',
-            node: callee.object.property,
+            node: calleeObject.property,
+            suggest: [
+              {
+                messageId: 'suggestRemoveFocus',
+                fix(fixer) {
+                  return [
+                    fixer.removeRange(
+                      calleeObject.property.type ===
+                        AST_NODE_TYPES.Identifier &&
+                        calleeObject.property.name === 'only'
+                        ? [calleeObject.object.range[1], calleeObject.range[1]]
+                        : [calleeObject.range[1], callee.range[1]],
+                    ),
+                  ];
```

Comment:
this was a bit hard to follow

```suggestion
                  if (
                    calleeObject.property.type === AST_NODE_TYPES.Identifier &&
                    calleeObject.property.name === 'only'
                  ) {
                    return fixer.removeRange([
                      calleeObject.object.range[1],
                      calleeObject.range[1],
                    ]);
                  }

                  return fixer.removeRange([
                    calleeObject.range[1],
                    callee.range[1],
                  ]);
```